### ISSUE TITLE
After Update to 2.1.0 .dat File is Missing in Flatpak on Linux

### ISSUE THREAD

**Issue by 4theEmpire** [k4zmu2a/SpaceCadetPinball#222]
Hi,
After the Flatpak Update today to Version 2.1.0 the Game is missing the .dat File. The former Version was running perfectly as Flatpak on opensuse Tumbleweed.
Best Regards

**Comment by WrongProblem** [comment]
Hello

this is not a software problem. I tried to downgrade to previous version and flatpak warned me that

Error: Wrong size for extra data https://archive.org/26/items/pinball_202007/Pinball.zip

so my guess is that after the great archive.org heist, file paths were changed on archive.org and the software can't find the original data files (these files can't be on Github for legal reasons) in their previous place. 

I have duckduckgoed the file for ya:

https://archive.org/details/pinball_202007

download the Pinball.zip file, unzip it (I mean the files, not the Pinball directory) into

~/.var/app/com.github.k4zmu2a.spacecadetpinball/data/SpaceCadetPinball

and it will make Pinball run again.

Update: opened https://github.com/flathub/com.github.k4zmu2a.spacecadetpinball/issues/31
